cfme/tests/configure/test_access_control.py

```diff
@@ -1590,7 +1589,24 @@ def test_tenant_unique_automation_domain_name_on_parent_level():
         initialEstimate: 1/2h
         startsin: 5.5
     """
-    pass
+    domain_name = fauxfactory.gen_alphanumeric()
+    domain1 = appliance.collections.domains.create(name=domain_name, enabled=True)
+    msg = "Name has already been taken"
+
+    with pytest.raises(Exception, match=msg):
+        domain2 = appliance.collections.domains.create(name=domain_name, enabled=True)
+
+    domain1.delete()
+
+    @request.addfinalizer
+    def _delete_domain():
+        if domain1.exists:
+            domain1.delete()
+        try:
```

Comment:
Q: why this exception handling? I don't think it will try to delete if not exist.